### DISCUSSION TITLE
Cannot push next.js project to github because large .next/cache files

### DISCUSSION THREAD

**Discussion by c2bsolutions** [community/community#36682] [Other Feature Feedback, Questions, & Ideas]
Hello,

I am having issues pushing a next js project to github due to the below error. I have tried ignoring and deleting the files and using git-lfs to track the files to avoid the error, but all my attempts have been unsuccessful. I found this discussion with the same issue but it has gone unresolved -> https://github.com/vercel/next.js/discussions/29233

Does anyone have a resolution for this error?

remote: warning: File .next/cache/webpack/client-development/index.pack.old is 73.64 MB; this is larger than GitHub's recommended maximum file size of 50.00 MB
remote: warning: File .next/cache/webpack/client-development/12.pack is 61.65 MB; this is larger than GitHub's recommended maximum file size of 50.00 MB
remote: warning: File .next/cache/webpack/client-development/9.pack is 63.60 MB; this is larger than GitHub's recommended maximum file size of 50.00 MB
remote: warning: File .next/cache/webpack/client-production/0.pack is 88.83 MB; this is larger than GitHub's recommended maximum file size of 50.00 MB
remote: warning: File .next/cache/webpack/client-development/index.pack is 73.64 MB; this is larger than GitHub's recommended maximum file size of 50.00 MB
remote: warning: File .next/cache/webpack/client-development/21.pack is 62.56 MB; this is larger than GitHub's recommended maximum file size of 50.00 MB
remote: warning: File .next/cache/webpack/client-development/7.pack is 66.98 MB; this is larger than GitHub's recommended maximum file size of 50.00 MB
remote: warning: File .next/cache/webpack/client-development/index.pack is 73.64 MB; this is larger than GitHub's recommended maximum file size of 50.00 MB
remote: warning: File .next/cache/webpack/client-development/index.pack.old is 73.64 MB; this is larger than GitHub's recommended maximum file size of 50.00 MB
remote: warning: File .next/cache/webpack/client-production/9.pack is 55.30 MB; this is larger than GitHub's recommended maximum file size of 50.00 MB
remote: warning: File .next/cache/webpack/client-development/25.pack is 62.44 MB; this is larger than GitHub's recommended maximum file size of 50.00 MB
remote: error: Trace: d4d828f15f1df9a3c905902455c7dc946017c5d660254b4577da082475068581
remote: error: See http://git.io/iEPt8g for more information.
remote: error: File .next/cache/webpack/client-development/28.pack is 124.65 MB; this exceeds GitHub's file size limit of 100.00 MB
remote: error: File .next/cache/webpack/client-development/1.pack is 155.89 MB; this exceeds GitHub's file size limit of 100.00 MB
remote: error: File .next/cache/webpack/client-production/0.pack is 144.56 MB; this exceeds GitHub's file size limit of 100.00 MB
remote: error: File .next/cache/webpack/client-development/0.pack is 325.31 MB; this exceeds GitHub's file size limit of 100.00 MB
remote: error: GH001: Large files detected. You may want to try Git Large File Storage - https://git-lfs.github.com/.

Thanks in Advance!

**Comment by ossanna16** [comment]
Hey there @c2bsolutions and thank you for asking a good question! I'm glad you found our community 🙂
